This patch will update the pre-generated rscryutil man page from the release
tarball so we don't need to depend on dev-python/docutils.

https://github.com/rsyslog/rsyslog/issues/73

diff -rupN old/rsyslog-7.6.3/tools/rscryutil.1 new/rsyslog-7.6.3/tools/rscryutil.1
--- old/rsyslog-7.6.3/tools/rscryutil.1	2013-10-29 16:31:21.000000000 +0100
+++ new/rsyslog-7.6.3/tools/rscryutil.1	2014-05-03 20:41:46.143825094 +0200
@@ -1,4 +1,4 @@
-.\" Man page generated from reStructeredText.
+.\" Man page generated from reStructuredText.
 .
 .TH RSCRYUTIL 1 "2013-04-15" "" ""
 .SH NAME
@@ -31,12 +31,16 @@ level margin: \\n[rst2man-indent\\n[rst2
 .in \\n[rst2man-indent\\n[rst2man-indent-level]]u
 ..
 .SH SYNOPSIS
+.INDENT 0.0
+.INDENT 3.5
 .sp
 .nf
 .ft C
 rscryutil [OPTIONS] [FILE] ...
 .ft P
 .fi
+.UNINDENT
+.UNINDENT
 .SH DESCRIPTION
 .sp
 This tool performs various operations on encrypted log files.
@@ -44,46 +48,46 @@ Most importantly, it provides the abilit
 .SH OPTIONS
 .INDENT 0.0
 .TP
-.B \-d,  \-\-decrypt
+.B \-d\fP,\fB  \-\-decrypt
 Select decryption mode. This is the default mode.
 .TP
-.BI \-W,  \-\-write\-keyfile \ <file>
+.BI \-W\fP,\fB  \-\-write\-keyfile \ <file>
 Utility function to write a key to a keyfile. The key can be obtained
 via any method.
 .TP
-.B \-v,  \-\-verbose
+.B \-v\fP,\fB  \-\-verbose
 Select verbose mode.
 .TP
-.B \-f,  \-\-force
+.B \-f\fP,\fB  \-\-force
 Forces operations that otherwise would fail.
 .TP
-.BI \-k,  \-\-keyfile \ <file>
+.BI \-k\fP,\fB  \-\-keyfile \ <file>
 Reads the key from <file>. File _must_ contain the key, only, no headers
 or other meta information. Keyfiles can be generated via the
 \fI\-\-write\-keyfile\fP option.
 .TP
-.BI \-p,  \-\-key\-program \ <path\-to\-program>
+.BI \-p\fP,\fB  \-\-key\-program \ <path\-to\-program>
 In this mode, the key is provided by a so\-called "key program". This program
 is executed and must return the key to (as well as some meta information)
 via stdout. The core idea of key programs is that using this interface the
 user can implement as complex (and secure) method to obtain keys as
 desired, all without the need to make modifications to rsyslog.
 .TP
-.BI \-K,  \-\-key \ <KEY>
+.BI \-K\fP,\fB  \-\-key \ <KEY>
 TESTING AID, NOT FOR PRODUCTION USE. This uses the KEY specified
 on the command line. This is the actual key, and as such this mode
 is highly insecure. However, it can be useful for intial testing
 steps. This option may be removed in the future.
 .TP
-.BI \-a,  \-\-algo \ <algo>
+.BI \-a\fP,\fB  \-\-algo \ <algo>
 Sets the encryption algorightm (cipher) to be used. See below
 for supported algorithms. The default is "AES128".
 .TP
-.BI \-m,  \-\-mode \ <mode>
+.BI \-m\fP,\fB  \-\-mode \ <mode>
 Sets the ciphermode to be used. See below for supported modes.
 The default is "CBC".
 .TP
-.BI \-r,  \-\-generate\-random\-key \ <bytes>
+.BI \-r\fP,\fB  \-\-generate\-random\-key \ <bytes>
 Generates a random key of length <bytes>. This option is
 meant to be used together with \fI\-\-write\-keyfile\fP (and it is hard
 to envision any other valid use for it).
@@ -97,7 +101,7 @@ multiple operations mode are set on the
 unpredictable.
 .SS decrypt
 .sp
-The provided log files are decrypted. Note that the \fI.encinfo\fP side files
+The provided log files are decrypted. Note that the \fI\&.encinfo\fP side files
 must exist and be accessible in order for decryption to to work.
 .SS write\-keyfile
 .sp
@@ -198,5 +202,4 @@ LGPLv2.
 .SH AUTHOR
 Rainer Gerhards <rgerhards@adiscon.com>
 .\" Generated by docutils manpage writer.
-.\" 
 .
